inc/namespace.php

```diff
@@ -3,11 +3,31 @@
 namespace Altis\Local_Server;
 
 use function Altis\get_config;
+use function Altis\get_environment_type;
 
 /**
  * Configure environment for local server.
  */
 function bootstrap() {
+	if ( get_environment_type() === 'local' ) {
```

Comment:
This code is only run if local server is in use so you don't need this check here, see the `wp-config.php` and `load.php` to see how that's achieved:

https://github.com/humanmade/altis-local-server/blob/master/load.php